### ISSUE TITLE
Add a new attribute to avoid diff

### ISSUE THREAD

**Issue by corbel88** [replaysMike/AnyDiff#6]
Hi,
Would it be possible to have a new attribute to avoid diff? I'm using the JSON serializer and I cannot use the dedicated attribute due to this.
Thanks

**Comment by replaysMike** [comment]
This is now included as of release version 1.0.61

**Comment by replaysMike** [comment]
A list of custom attributes to treat as ignore attributes can now be passed in via DiffOptions.
How to use:

```csharp
var object1 = new CustomAttributeObject(1, "A string");
var object2 = new CustomAttributeObject(1, "A different string");
var diffOptions = new DiffOptions(new List<object> { typeof(System.ComponentModel.DescriptionAttribute) });
var diff = object1.Diff(object2, ComparisonOptions.All, diffOptions);
Assert.AreEqual(0, diff.Count);
```
